How-To/Connect-to-AWS-Resources.md

```diff
@@ -48,9 +48,14 @@ s3 = boto3.client(
 
 ## Athena
 
-### R (Small Queries)
+### R
 
-You can use the `noctua` R package to pull small amounts of data from Athena. This package uses the `paws` R package to directly grab query results, but it can be significantly slower and more memory-intensive than the JDBC connector. The benefit of `noctua` is that is can cache query results for faster reload of the same data. See [here](https://dyfanjones.github.io/noctua/articles/aws_athena_query_caching.html) for more details.
+You can use the `noctua` R package to pull small amounts of data from Athena. `noctua` has two useful options for speeding up queries: [caching](https://dyfanjones.github.io/noctua/articles/aws_athena_query_caching.html) and [unload](https://dyfanjones.github.io/noctua/articles/aws_athena_unload.html).
```

Comment:
**[Nitpick, non-blocking]** Do we still only recommend noctua in the case of "small amounts of data", or is that out of date per the PR description?

```suggestion
You can use the `noctua` R package to pull data from Athena. `noctua` has two useful options for speeding up queries: [caching](https://dyfanjones.github.io/noctua/articles/aws_athena_query_caching.html) and [unload](https://dyfanjones.github.io/noctua/articles/aws_athena_unload.html).
```

Reply:
That's out of date! Good catch